### ISSUE TITLE
JSON decode in PHP

### ISSUE THREAD

**Issue by DeanHickerson** [DeanHickerson/Apprise#4]
Need to get JSON decode to work in PHP. Might need to turn turn off `magic_quotes_gpc`. I don't know if I really want to turn it off at the PHP settings level, so it might be best to use this code to turn it off at runtime.
```
if (get_magic_quotes_gpc()) {
    $process = array(&$_GET, &$_POST, &$_COOKIE, &$_REQUEST);
    while (list($key, $val) = each($process)) {
        foreach ($val as $k => $v) {
            unset($process[$key][$k]);
            if (is_array($v)) {
                $process[$key][stripslashes($k)] = $v;
                $process[] = &$process[$key][stripslashes($k)];
            } else {
                $process[$key][stripslashes($k)] = stripslashes($v);
            }
        }
    }
    unset($process);
}
```

**Comment by DeanHickerson** [comment]
Screw it. Lets use this:
```
function append_to_the_log($data) {
        $curLog = file_get_contents('log.json');
        $logPrep = substr($curLog, 0, -1);
        $logReady = $logPrep.','.$data.']';
        $fp = fopen('log.json', 'w');
        fwrite($fp, $logReady);
        fclose($fp);
}
```

**Comment by DeanHickerson** [comment]
Fixed with Commit: 4d1b599acf940193313dd815c5f067a3759e7627
